### PR TITLE
Feature/multiuuid post

### PR DESCRIPTION
**Pull request by kovacshuni**
able to retrieve items by HTTP POST including an ["array", "of", "uuids"]

### CODE REVIEW

File: src/main/java/com/ft/universalpublishing/documentstore/resources/DocumentResource.java [hunk 1]
```diff
@@ -58,6 +60,23 @@ public final Object getFromCollectionByUuids(
     return handlerChain.execute(context);
   }
 
+  @POST
+  @Timed
+  @Path("/{collection}/getMultiple")
```
Comment (danielcopaciu) [inline src/main/java/com/ft/universalpublishing/documentstore/resources/DocumentResource.java]: This path does not look very restful. First, it says get when you actually do a post. Does POST /{collection} conflict with another endpoint?

Comment (kovacshuni) [reply src/main/java/com/ft/universalpublishing/documentstore/resources/DocumentResource.java]: No it doesn't conflict. But then one would think it's an update, like in a normal CRUD API. We have an identical PUT endpoint for something like that.

Comment (danielcopaciu) [reply src/main/java/com/ft/universalpublishing/documentstore/resources/DocumentResource.java]: Maybe we can use /{collection}/set

Comment (khatton-ft) [reply src/main/java/com/ft/universalpublishing/documentstore/resources/DocumentResource.java]: Yeah, the whole point is that this has to be a POST request if you don't want to limit the number of UUIDs that can be passed in a single request. (The other option would be to batch up the UUIDs in the client and call the existing GET endpoint multiple times for those few cases where there are more UUIDs than will fit in a single request).

I do like the idea of just POSTing to the same URL as the GET, though. Maybe with a query param e.g. `POST /{collection}?mget=true` which gives a hint all round that this is a multiple-GET request.

Comment (sjwells) [reply src/main/java/com/ft/universalpublishing/documentstore/resources/DocumentResource.java]: I like Keith's suggestion of using a query parameter

Comment (kovacshuni) [reply src/main/java/com/ft/universalpublishing/documentstore/resources/DocumentResource.java]: using `POST /{collection}?mget=true`

**Pull request merged**